src/loaders/api/api.js

```diff
@@ -50,7 +50,7 @@ export function setAPI (agentIdentifier, nr, forceDrain) {
   var spaPrefix = prefix + 'ixn-'
 
   // Setup stub functions that queue calls for later processing.
-  mapOwn(asyncApiFns, function (num, fnName) {
+  asyncApiFns.forEach(fnName => {
```

Comment:
If we are going to arbitrarily rework using `mapOwn`, we should remove it completely from the file.